src/QsCompiler/Core/Dependencies.fs

```diff
@@ -238,6 +239,12 @@ type BuiltIn =
     static member EnableTestingViaName =
         { FullName = { Name = "EnableTestingViaName"; Namespace = BuiltIn.DiagnosticsNamespace }; Kind = Attribute }
 
+    static member ReadCycleCounter =
+        {
+            FullName = { Name = "ReadCycleCounter"; Namespace = BuiltIn.DiagnosticsNamespace }
```

Comment:
I think this should be in the Core namespace or the intrinsic namespace. Diagnostic feels odd to me, though perhaps there is a logic I'm missing.

Reply:
I suggest to put it into a new namespace Microsoft.Quantum.LlvmIntrinsics.